src/main/java/jenkins/plugins/http_request/HttpRequestGlobalConfig.java

```diff
@@ -78,10 +82,18 @@ public static HttpRequestGlobalConfig get() {
         return GlobalConfiguration.all().get(HttpRequestGlobalConfig.class);
     }
 
+    /**
+     * @deprecated removed without replacement
+     */
+    @Deprecated
     public List<BasicDigestAuthentication> getBasicDigestAuthentications() {
```

Comment:
Usage of deprecated method still present in tests

Reply:
Fixed.